test/_assert_utils.py

```diff
@@ -0,0 +1,10 @@
+"""This is a temporary module and should be removed as soon as torch.testing.assert_equal is supported."""
```

Comment:
Maybe add a "TODO: remove this when..." comment so that we have a match when doing `git grep TODO`